shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -73,6 +73,12 @@ - (instancetype)initWithFrame:(NSRect)frame
                                                                           layer:self.layer];
     _resizeSynchronizer =
         [[FlutterResizeSynchronizer alloc] initWithDelegate:_resizableBackingStoreProvider];
+    NSString* applicationName =
```

Comment:
You're setting this string, but never using it.

Also, per previous discussion you should be allowing clients to set the view name (or at least putting a TODO here indicating that that will be needed in the future.)